### PR TITLE
Disable some of the kubeflow repos' trigger

### PR DESCRIPTION
**Pull request by PatrickXYS**
Given the fact that some of the kubeflow repos exist two bot respond to `/retest`, `/test all`, etc. This is caused by some of the repos migrated to AWS CI, but still rely on upstream prow/tide managed PR.

E.g, [manifests PR](https://github.com/kubeflow/manifests/pull/1603#issuecomment-722539135)

Thus, disabling some of the kubeflow repos' trigger plugin to avoid two bots multiple response and save developer's confusions

/cc @jlewi @cblecker

### CODE REVIEW

**Comment by MushuEE** [comment]
Would it be helpful to have an `exclude_repos` option?

**Comment by PatrickXYS** [comment]
> Would it be helpful to have an exclude_repos option?

That would be great if we have, but I don't think trigger support this feature yet, someone can correct me if I'm wrong

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -19,7 +19,26 @@ triggers:
   - containerd/containerd
   join_org_url: "https://github.com/containerd/project/blob/master/MAINTAINERS"
 - repos:
```
Comment (jlewi) [inline config/prow/plugins.yaml:21]: @PatrickXYS Lets add a comment here explaining why the default is no longer "on" for new repos.

Can we also file an issue to track updating the new repository setup guide
https://github.com/kubeflow/community/blob/master/repository-setup.md

/cc @Bobgy

Comment (PatrickXYS) [reply config/prow/plugins.yaml:21]: I added comments to explain the reason, also, file issue here to keep track https://github.com/kubeflow/community/issues/450

**Comment by Bobgy** [comment]
/lgtm
Thanks!

**Comment by Bobgy** [comment]
@PatrickXYS I'm not sure about the change, in fact. Will k8s tide continue to work on those repos? (Tide is responsible for merging PRs)
If not, has @aws-kf-ci-bot been configured to handle tide actions? (does it have permissions?)

**Comment by PatrickXYS** [comment]
> I'm not sure about the change, in fact. Will k8s tide continue to work on those repos? (Tide is responsible for merging PRs)

What I change here is about Prow plugin - [trigger](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/trigger), and [Tide](https://github.com/kubernetes/test-infra/tree/master/prow/tide) is one of the core component in Prow.

So I think Tide should still work as before, and k8s-ci-bot will continuously work on those repos. 

I'm okay to start with one repo if we're not sure

/cc @Bobgy @jlewi

**Comment by Bobgy** [comment]
SGTM


@PatrickXYS because there are no people we can ask for quick approval if the change broke some repos. The risk is high, so shall we reduce to just one repo first?

**Comment by jlewi** [comment]
So  the plan is to merge #19897 first to test with a single repo and then if that looks good merge this one?

**Comment by PatrickXYS** [comment]
> So the plan is to merge #19897 first to test with a single repo and then if that looks good merge this one?

Yes, so the #19897 will be able to confirm if there's any risk in disabling repo's trigger, after confirmation, we'll go ahead with other repos.

**Comment by PatrickXYS** [comment]
We started experiment on this PR https://github.com/kubernetes/test-infra/pull/19897

After it merged, we expect kubeflow/kfctl only exist one bot to respond `/test ?`, 

and I experimented here https://github.com/kubeflow/kfctl/pull/247#issuecomment-728388424, it's working properly.

I'll go ahead with other repos in this one single PR

/cc @jlewi @Bobgy

**Comment by PatrickXYS** [comment]
```
{"component":"unset","file":"prow/cmd/checkconfig/main.go:79","func":"main.reportWarning","level":"warning","msg":"the following repos have jobs configured but do not have the trigger plugin enabled: repo: kubeflow/pytorch-operator","severity":"warning","time":"2020-11-17T13:17:30Z"}
```

This is the error message, but that's exactly what we want to do.

I assume this is because pytorch-operator still have postsubmit job configured https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/kubeflow-postsubmits.yaml#L57-L69

```
  kubeflow/pytorch-operator:
  - name: kubeflow-pytorch-operator-postsubmit
    cluster: kubeflow
    labels:
      preset-service-account: "true"
    spec:
      containers:
      - image: gcr.io/kubeflow-ci/test-worker:latest
        imagePullPolicy: Always

    annotations:
      testgrid-dashboards: sig-big-data
      description: Postsubmit kubeflow/pytorch-operator.
```

Will try to clean up this as well

**Comment by PatrickXYS** [comment]
Can you help on adding / ok-to-test and I'll wait to see if test can pass? @jlewi

**Comment by PatrickXYS** [comment]
Friendly ping @jlewi  @cblecker

**Comment by PatrickXYS** [comment]
```
ERROR: /home/prow/go/src/github.com/kubernetes/test-infra/config/tests/testgrids/BUILD.bazel:25:1: Executing genrule //config/tests/testgrids:testgrid_config failed (Exit 1)
2020/11/20 15:31:49 FAIL: could not write config: 1 error occurred:
	* configuration error for (TestGroup) kubeflow-pytorch-operator-postsubmit: Each Test Group must be referenced by at least 1 Dashboard Tab.
```

I see, maybe I need to clean up testgird config for pytorch as well

**Pull request merged**